.classpath

```diff
@@ -11,7 +11,7 @@
 			<attribute name="javadoc_location" value="jar:platform:/resource/2017-2706-Robot-Code/lib/non-robot-libs/google/docs.zip!/"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="var" path="USERLIBS_DIR/CTRLib.jar"/>
-	<classpathentry kind="var" path="USERLIBS_DIR/navx_frc.jar"/>
+	<classpathentry kind="lib" path="build/jars/navx_frc.jar"/>
+	<classpathentry kind="lib" path="build/jars/CTRLib.jar"/>
```

Comment:
This will not deploy correctly, it needs to be exactly the same as it was before.